modules/apps/29-fee/client/cli/tx.go

```diff
@@ -97,3 +97,28 @@ func NewPayPacketFeeAsyncTxCmd() *cobra.Command {
 
 	return cmd
 }
+
+// NewRegisterCounterpartyAddress returns the command to create a MsgRegisterCounterpartyAddress
+func NewRegisterCounterpartyAddress() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "register-counterparty [address] [counterparty-address] [channel-id]",
+		Short:   "Register a counterparty relayer address on a given channel.",
+		Long:    strings.TrimSpace(`Register a counterparty relayer address on a given channel.`),
+		Example: fmt.Sprintf("%s tx ibc-fee register-counterparty cosmos1rsp837a4kvtgp2m4uqzdge0zzu6efqgucm0qdh cosmoss1sp921a4tttgpln6rqhdqe0zzu6efqgucm0qdh channel-0", version.AppName),
+		Args:    cobra.ExactArgs(3),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			msg := types.NewMsgRegisterCounterpartyAddress(args[0], args[1], args[2])
```

Comment:
Does it make sense to validate `args[0]` and `args[1]` to make sure they are valid addresses?

Reply:
`msg.ValidateBasic` is called on the message which will do validation on the strings as appropriate